### PR TITLE
Update scopes documentation

### PR DESCRIPTION
**Pull request by torcolvin**
Fix some API documentation.

### CODE REVIEW

File: docs/api/components/schemas.yaml [hunk 1]
```diff
@@ -1165,7 +1165,9 @@ Database:
       description: The name of the database.
       type: string
     sync:
-      description: The Javascript function that newly created documents are ran through for the _default scope and collection.
+      description: The Javascript function that newly created documents are ran through for the default scope and collection.
+
+        If `scopes` parameter is set, this is ignored.
```
Comment (bbrks) [inline docs/api/components/schemas.yaml:1170]: Should we add SG config validation to cover this case - to explicitly disallow it?

File: docs/api/components/schemas.yaml [hunk 2]
```diff
@@ -1203,9 +1205,11 @@ Database:
       default: 16
     import_filter:
       description: |-
-        This is the function that all imported documents in the _default scope and collection are ran through in order to filter out what to import and what not to import. This allows you to control what is made available to Couchbase Mobile clients. If it is not set, then no documents are filtered when imported.
+        This is the function that all imported documents in the default scope and collection are ran through in order to filter out what to import and what not to import. This allows you to control what is made available to Couchbase Mobile clients. If it is not set, then no documents are filtered when imported.
 
         `import_docs` must be true to make this field applicable.
+
+        If `scopes` parameter is set, this is ignored.
```
Comment (bbrks) [inline docs/api/components/schemas.yaml:1212]: Ditto

File: docs/api/components/schemas.yaml (hunk 1 above)
Comment (torcolvin) [reply docs/api/components/schemas.yaml:1170]: Already done in https://github.com/couchbase/sync_gateway/blob/ae024a247c0ee62a5598121843e9904084ac15e1/rest/config.go#L852

File: docs/api/components/schemas.yaml (hunk 2 above)
Comment (torcolvin) [reply docs/api/components/schemas.yaml:1212]: Already done in https://github.com/couchbase/sync_gateway/blob/ae024a247c0ee62a5598121843e9904084ac15e1/rest/config.go#L855

**Review by bbrks — APPROVED** [review]

**Pull request merged**